Example_SwiftUI/Example_SwiftUI/Example_SwiftUIApp.swift

```diff
@@ -25,6 +25,8 @@ class AppDelegate: NSObject, UIApplicationDelegate {
         _: UIApplication,
         didFinishLaunchingWithOptions _: [UIApplication.LaunchOptionsKey: Any]? = nil
     ) -> Bool {
+        // Remove comment below if you want to specific which features that will be removed. and don't forget to comment DebugSwift.setup()
+        // DebugSwift.hide(features: [.interface,.app,.resources,.performance])
```

Comment:
Can I propose more one little change?
If you change:

DebugSwift.hide(features: [.interface,.app,.resources,.performance])

to 

DebugSwift.setup(hideFeatures: [.interface, .app, .resources, .performance])

It will be clearer for whoever is implementing it :)

Reply:
Thanks for your advice @MaatheusGois , ok I'll change function name to DebugSwift.setup(hideFeatures: [.interface, .app, .resources, .performance]) :)